catroid/src/main/java/org/catrobat/catroid/ui/SettingsActivity.java

```diff
@@ -368,6 +385,14 @@ public static void setAutoCrashReportingEnabled(Context context, boolean value)
 		SharedPreferences.Editor editor = getSharedPreferences(context).edit();
 		editor.putBoolean(SETTINGS_CRASH_REPORTS, value);
 		editor.commit();
+
+		if (!value) {
+			CrashReporter.disableCrashlytics();
+			Log.e(TAG, "Crashlytics Disabled");
+		} else {
+			CrashReporter.initialize(context);
+			Log.e(TAG, "Crashlytics Enabled");
```

Comment:
You are right. We already did the logging at initialize() method so I'll be removing it from here.